### ISSUE TITLE
RadioTile Arrow key navigation not working in Firefox

### ISSUE THREAD

**Issue by cricciuto** [carbon-design-system/carbon#6496]
## What package(s) are you using?
- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

When output many RadioTiles in a TileGroup I expect to be able to use the arrow keys on my keyboard to navigation between the options on Firefox.
This works in Chrome but not Firefox. Did not try other browsers.

> Is this issue related to a specific component?

Only tried with RadioTile in a TileGroup

> What did you expect to happen? What happened instead? What would you like to
> see changed?

Keyboard navigation between RadioTile options to work like in Chrome

> What browser are you working in?

Firefox v78 and Firefox Dev Edition v79

> What version of the Carbon Design System are you using?

carbon-components: "^10.10.1"
carbon-components-react: "7.10.3"

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

An IBM product

## Steps to reproduce the issue

1. Try to use RadioTiles (in on TileGroup) with keyboard arrows
Can even just use the example on the [carbondesignsystem website](https://www.carbondesignsystem.com/components/tile/usage/) in Firefox

**Comment by cricciuto** [comment]
So I was playing around with the example linked to in the description and the keyboard arrow navigate does work in one case but not well.

Using the linked example if you select the "Radio tile" example and then tab the way to the group of RadioTiles it will highlight the first item (with the second/middle is selected by default) but you will not be able to use the arrow keys here. If you hit the `TAB` key once more then you will be able to use the keyboard arrow keys to navigate the RadioTiles.

Another way to show the issue is to click on a RadioTile to select it and put focus on it (with the blue highlight). The arrow keys for navigation won't work but if you perform a `SHIFT + TAB` keyboard-combo you will now be able to use the arrow keys to navigation the RadioTiles.
